content/docs/concepts/security/index.md

```diff
@@ -223,86 +223,90 @@ origins:
 
 Defines what is the principal from the authentication. By default, this will be the peer's principal (and if peer authentication is not applied, it will be left unset). Policy writers can choose to overwrite it with USE_ORIGIN. In future, we will also support *conditional-binding* (e.g USE_PEER when peer is X, otherwise USE_ORIGIN)
 
-## Role-Based Access Control (RBAC)
+## Istio Authorization
```

Comment:
Headings like this should only capitalize the first letter.

Note that calling "Istio Authorization" is not great, the word Istio is redundant. We generally don't say "Istio Circuit Breaking", "Istio Mixer Adapters", "Istio Fault Injection". Just "Authorization" is sufficient IMO

Reply:
Makes sense. I changed "Istio Authorization" to "Authorization".